changelog.md

```diff
@@ -1,7 +1,53 @@
+* Introducing trials: Each config/seed/budget/instance calculation is a trial.
+* The configuration chooser is integrated into the SMBO object now. Therefore, SMBO finally implements an ask-tell interface now.
+* Facades are redesigned so that they accept instantiated components directly. If a component is not passed, a default component is used, which is specified for each facade individually in the form of static methods. You can use those static methods directly to adapt a component to your choice.
+* A lot of API changes and renamings (e.g., RandomConfigurationChooser -> RandomDesign, Runhistory2EPM -> RunHistoryEncoder).
+* Ambiguous variables are renamed and unified across files.
+* Dependencies of modules are reduced drastically.
+* We incorporated Pynisher 1.0, which ensures limitations cross-platform.
+* We incorporated ConfigSpace 0.6, which simplified our examples.
+* Examples and documentation are completely reworked. Examples use the new ConfigSpace, and the documentation is adapted to version 2.0.
+* Transparent target function signatures: SMAC checks now explicitly if an argument is available (the required arguments are now specified in the intensifier). If there are more arguments that are not passed by SMAC, a warning is raised.
+* Instances, seeds, and budgets can be set to none now. However, mixing none and non-none will throw an exception.
+* Components implement a ``meta`` property now, all of which describe the initial state of SMAC. The facade collects all metadata and saves the initial state of the scenario.
+* Improved multi-objective in general: RunHistory (in addition to RunHistoryEncoder) both incorporates the multi-objective algorithm. In other words, if the multi-objective algorithm changes the output, it directly affects the optimization process.
+* Configspace is saved in json only
+* StatusType is saved as integer and not as dict anymore
+* We changed the behavior of continuing a run:
+    * SMAC automatically checks if a scenario was saved earlier. If there exists a scenario and the initial state is the same, SMAC automatically loads the previous data. However, continuing from that run is not possible yet.
+    * If there was a scenario earlier, but the initial state is different, then the user is asked to overwrite the run or to still continue the run although the state is different (Note that this only can happen if the name specified in the scenario is the same). Alternatively, an `old` to the old run is added.
+    * The initial state of the SMAC run also specifies the name (if no name in the scenario is specified). If the user changes something in the code base or in the scenario, the name and, therefore, the save location automatically changes.
```

Comment:
"If the user changes something in the code base or in the scenario, the name and, "
something is missing here?

Reply:
no it's not